src/java/org/apache/cassandra/utils/reflection/ReflectionUtils.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.utils.reflection;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+
+public class ReflectionUtils
+{
+    private ReflectionUtils()
+    {
+
+    }
+
+    public static Field getModifiersField() throws NoSuchFieldException
```

Comment:
Yup, I suggested this also in my branch for jdk 17 https://github.com/apache/cassandra/commit/4daf0b149bc1524dfb2212ab958f4d45feb79577#diff-929245717cd8c37ec78fb97bddc5b6839b2430c3e32f31f30e4aad5fcd580c71R376-R404 I will put a note to get rid of this in my WIP branch and use this utility class on next rebase :-) Thanks

Reply:
reverted for now as this is for JDK 17